src/test/java/teammates/storage/api/FeedbackResponseMonitorDbTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.storage.api;
+
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.FeedbackResponseRecordAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.test.BaseComponentTestCase;
+
+/**
+ * SUT: {@link FeedbackResponseMonitorDb}.
+ */
+public class FeedbackResponseMonitorDbTest extends BaseComponentTestCase {
+    FeedbackResponseMonitorDb db = new FeedbackResponseMonitorDb();
+    long[] countTestData = new long[] { 105, 105, 105, 105, 105 };
```

Comment:
Array? Why not ArrayList?

Reply:
I dont think the test data needs to be dynamic

Reply:
Use of List is not about dynamics; it's about abstraction. Use of List prolly yield a similar performance compared to array but is more suitable in many more cases.

Reply:
Why is List suitable here?